src/components/PageDefinitions.js

```diff
@@ -213,6 +214,11 @@ class PageDefinitions extends AbstractPageDefinitions {
     })
   }
 
+  onDropRejected = files => {
+    const fileNames = files.map(file => file.name).join(', ')
+    this.props.dispatch(uiNotificationNew({ type: 'danger', message: `Could not load: ${fileNames}`, timeout: 5000 }))
```

Comment:
The message should be more informative. It is not that we tried to load and could not it is that the type of the file is not supported.